app/webpacker/stylesheets/patterns/_tabs.scss

```diff
@@ -0,0 +1,4 @@
+.app-tabs__list .govuk-tabs__list-item--selected {
```

Comment:
This PR doesn't have any tabs and we have styles from the design system

Reply:
Removed.